src/middlewares/Mock.spec.ts

```diff
@@ -0,0 +1,128 @@
+import {Client, IResponse} from "../Client";
+import {Mock} from "./Mock";
+describe("Mock", () => {
+    it("is defined", () => {
+        expect(Mock).toBeDefined();
+    });
+
+    describe("constructor()", () => {
+        it("accepts IHandler array in constructor", () => {
+            const obj = new Mock([
+                {
+                    delay: 0,
```

Comment:
delay is optional, remove it. No need to clutter the test file. 

Also... is this test really checking what we want? Ideas?